src/components/shared/ListField.tsx

```diff
@@ -27,21 +28,56 @@ const ListField: FC<IListFieldProps> = ({
 
   function getSapCostValue(field: IForm):string {
     let sapValue = 0;
-
-    if (field.name === 'realizedCost') {
-      sapValue = field.sapCosts?.project_task_costs || 0;
+    if (field.name === 'realizedCostCurrentYear') {
+        const projectTaskCosts = Number(field.sapCurrentYear?.project_task_costs) || 0;
+        const productionTaskCosts = Number(field.sapCurrentYear?.production_task_costs) || 0;
+        sapValue = projectTaskCosts + productionTaskCosts;
+    }
+    else if (field.name === 'realizedCost') {
+      const projectTaskCosts = Number(field.sapCosts?.project_task_costs) || 0;
+      const productionTaskCosts = Number(field.sapCosts?.production_task_costs) || 0;
+      sapValue = projectTaskCosts + productionTaskCosts;
     }
     else if (field.name === 'comittedCost') {
-      sapValue = field.sapCosts?.project_task_commitments || 0;
+      const project_task_commitments = Number(field.sapCosts?.project_task_commitments) || 0;
+      const production_task_commitments = Number(field.sapCosts?.production_task_commitments) || 0;
+      sapValue = project_task_commitments + production_task_commitments;
     }
     else if (field.name === 'spentCost') {
-      const realizedCost = field.sapCosts?.project_task_costs || 0;
-      const comittedCost = field.sapCosts?.project_task_commitments || 0;
+      const projectTaskCosts = Number(field.sapCosts?.project_task_costs) || 0;
+      const productionTaskCosts = Number(field.sapCosts?.production_task_costs) || 0;
+      const project_task_commitments = Number(field.sapCosts?.project_task_commitments) || 0;
+      const production_task_commitments = Number(field.sapCosts?.production_task_commitments) || 0;
+      const realizedCost = projectTaskCosts + productionTaskCosts;
+      const comittedCost = project_task_commitments + production_task_commitments;
       sapValue = Number(comittedCost) + Number(realizedCost);
     }
     return Number(sapValue).toFixed(0);
   };
 
+  const showTooltip = (field: IForm) => {
+    if (field.name === 'realizedCostCurrentYear' || field.name === 'realizedCost' || field.name === 'comittedCost' || field.name === 'spentCost') {
+        return true; 
+    }
+  }
+
+  const renderTooltip = (field: IForm) => {
+    let tooltipLabel = '';
+    if (field.name === 'realizedCostCurrentYear') {
+      tooltipLabel = t('projectForm.realizedCostCurrentYearTooltipLabel')
+    }
+    else if (field.name === 'realizedCost') {
+      tooltipLabel = t('projectForm.realizedCostTooltipLabel')
+    }
+    else if (field.name === 'comittedCost') {
+      tooltipLabel = t('projectForm.comittedCostTooltipLabel')
+    }
+    else if (field.name === 'spentCost') {
+      tooltipLabel = t('projectForm.spentCostTooltipLabel')
+    }
+    return <Tooltip placement="top-start" className="list-field-tool-tip" >{tooltipLabel}</Tooltip>
+  }
```

Comment:
This could be refactored to not use the if else if structure to make it a bit more simple. For example:
```
const renderTooltip = (field: IForm) => {
    const tooltipLabels: { [key: string]: string } = {
      realizedCostCurrentYear: t('projectForm.realizedCostCurrentYearTooltipLabel'),
      realizedCost: t('projectForm.realizedCostTooltipLabel'),
      comittedCost: t('projectForm.comittedCostTooltipLabel'),
      spentCost: t('projectForm.spentCostTooltipLabel'),
    };
  
    const tooltipLabel = tooltipLabels[field.name] || '';
    return <Tooltip placement="top-start" className="list-field-tool-tip" >{tooltipLabel}</Tooltip>;
  };
```